mtags/src/main/scala-2/scala/meta/internal/pc/InferredMethodProvider.scala

```diff
@@ -177,6 +178,33 @@ final class InferredMethodProvider(
     } else None
   }
 
+  private def inferArgType(arg: Tree): String = {
```

Comment:
This won't work for lambda expression with short syntax, like:
```Scala
List(...).map(<<method>>)
```
right? But it also shouldn't call this method in such a case, right?

Reply:
Yeah, it will not be called in that case. This is strictly in cases where we want to calculate parameter types of the non existing function.